packages/happn-3/package.json

```diff
@@ -63,6 +63,7 @@
     "happn-db-provider-mongo": "1.1.8",
     "happn-random-activity-generator": "0.2.1",
     "jsprim": "^2.0.0",
+    "mocha": "^10.2.0",
```

Comment:
why is this here? mocha should be available in happn-commons-test (which is also required)